assets/base.css

```diff
@@ -3279,12 +3280,11 @@ details-disclosure > details {
 
   .scroll-trigger.animate--fade-in,
   .scroll-trigger.animate--slide-in {
-    transition: opacity var(--duration-extra-long) ease-out;
-    opacity: 0;
+    opacity: 0.1;
```

Comment:
It doesn't make much of a difference in practice, but I'm curious why we're using `0.1` instead of `0` here?

Reply:
It was something to test that Stu found out. It has an impact on performance scores. 

There is some details mentioned in here: https://www.debugbear.com/blog/opacity-animation-poor-lcp

Reply:
Oh, interesting! Sounds good to me. 👍

Reply:
Thanks for saving the day @kjellr 

Looks like having the opacity set to `1` on the element and applying the animation without the `forwards` fill-mode solved the issue 🤞 🙌

Reply:
I'm curious: Does the `0.1` opacity performance fix still work if the opacity is set to `0.01` instead? 

Something about the animations with 10% opacity just feels less smooth — I think it's because you can still see dark items really clearly _before_ they animate in. Here's an example in slow-motion. Note that the first frame you see is of the dark box on the left: 

https://user-images.githubusercontent.com/1202812/234870650-cecb2c59-962d-47c5-8ccc-93e5d5fa18e9.mp4

That kinda interrupts the impression of sequential left-to-right loading. If we're able to set the opacity to 1% instead of 10%, then the effect is more as intended: 

https://user-images.githubusercontent.com/1202812/234870915-3b37c904-61f9-4ab7-a3b1-5c9a5e725a8d.mp4